packages/react-storage/src/components/StorageBrowser/views/LocationActionView/hooks/__tests__/useDeleteActionView.spec.tsx

```diff
@@ -0,0 +1,353 @@
+import { renderHook, act } from '@testing-library/react-hooks';
```

Comment:
Trying to move away from usage of `'@testing-library/react-hooks'`, `renderHook` is available from `'@testing-library/react'`

Reply:
fixed